File: tests/fixtures/signatures/intra_function_duplicate.ts
export const completedResult = (raw: string): object => {
  try {
    return JSON.parse(raw);
  } catch {
    return failure({
      status: "failed",
      reason: "semantic_worker_bad_response",
      detail: "invalid worker response",
    });
  }

  if (!raw) {
    return failure({
      status: "failed",
      reason: "semantic_worker_bad_response",
      detail: "invalid worker response",
    });
  }
};

File: tests/fixtures/signatures/intra_function_duplicate.py
def completed_result(raw, stderr):
    try:
        payload = json.loads(raw)
    except json.JSONDecodeError:
        return failure(
            status="failed",
            reason="semantic_worker_bad_response",
            stderr=stderr,
            code=1,
        )

    if not isinstance(payload, dict):
        return failure(
            status="failed",
            reason="semantic_worker_bad_response",
            stderr=stderr,
            code=1,
        )

    return payload
